css-conditional-5/Overview.bs

```diff
@@ -336,4 +1241,40 @@ Additions since Level 4</h3>
 		<li>Added ''@when'' and ''@else''.
 		<li>Extended [=supports queries=] to express font capabilities
 			via ''font-tech()'' and ''font-format()''.
+		<li>Moved Container Queries from [[CSS-CONTAIN-3] to this specification.
+			(See also the [[CSS-CONTAIN-3#changes]] for more information
+			on the evolution of this feature.)
 	</ul>
+
+<h2 class="no-num" id="acknowledgments">Acknowledgments</h2>
```

Comment:
I think you should probably remove me and Elika from this list since editors shouldn't acknowledge themselves.

Reply:
Done